src/controllers/order.controller.ts

```diff
@@ -28,8 +31,25 @@ class OrderController {
     }
   };
 
-  public static getOrdersHandler: RequestHandler = async (req, res: ApiResponse, next) => {
+  public static getOrdersHandler: RequestHandler<
+    {},
+    {},
+    {},
+    {
+      reservationLogId: OrderLog['reservationLogId'];
+    }
```

Comment:
NITS: use RequestHandler would be ok.
https://www.programcreek.com/typescript/?api=express.RequestHandler